### PR TITLE
Added a winter coat for chef called "chef's freezer coat" to the chefdrobe

### PR DESCRIPTION
**Pull request by Alithsko**
…chefdrobe

Art by WraptorTek

## About the PR
Adds a variant of winter coat for chefs so that they can work better in the freezer, this will mostly prevent having to collect a generic winter coat at the start of the game

**Media**
Freezer Jacket 1
Freezer Jacket 2
Freezer Jacket 3
Freezer Jacket 4
Freezer Jacket 5

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

- [ X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
V1.0 Added chef's freezer coat to ChefDrobe

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml [hunk 1]
```diff
@@ -77,6 +77,17 @@
     sprite: Clothing/OuterClothing/WinterCoats/coatcentcom.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/WinterCoats/coatcentcom.rsi
+    
+    - type: entity
```
Comment (Visne) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml:81]: Please fix the indentation

**Review by Visne — CHANGES_REQUESTED** [review]

**Comment by AJCM-git** [comment]
This is missing `coatchef.rsi`

File: Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml (hunk 1 above)
Comment (Alithsko) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml]: I think I fixed it, it's strange as there were no extra spaces when I first commited this

**Comment by Alithsko** [comment]
> This is missing `coatchef.rsi`

My bad I think the sprites are in there now

**Review by Visne — APPROVED** [review]

**Pull request merged**